docs/source/irene.rst

```diff
@@ -161,7 +161,8 @@ Peak finding and matching of PMT and SiPM signals
 
 The peak finding and waveform slicing is arguably the most complex part of the **RWF** processing. The algorithm must be able to find two very different types of signals (S1 and S2), while accurately establishing the limits on those peaks to maintain the energy resolution capabilities of the detector.
 
-In order to optimize the peak search, PMT **CCWF**\ s are used as they have a higher sampling rate and therefore better time resolution. On top of that, these waveforms are PMT-summed to increase the signal-over-noise ratio [#]_. S1 and S2 signals are searched independently.
+In order to optimize the peak search, PMT **CCWF**\ s are used as they have a higher sampling rate and therefore better time resolution. The searches for S1 and S2 signals use slightly different waveforms. The search for S1 peaks uses the MAU-subtracted **CCWF**\ s. The PMT waveforms contain some low-frequency noise that can shift the PMT baseline locally by a small amount. The MAU (Moving Average Uindow) computes these small local variations of the baselines, which allows for a better peak finding algorithm. The parameter ``n_mau`` controls the size of this window. The S2 peak search uses the **CCWF**\ s directly.
```

Comment:
I wonder if we should define what is MAU before mention it, and not the other way around. Something like this: 

`"The search for S1 peaks uses the MAU *(Moving Average Uindow)* subtracted **CCWF**\ s. The PMT waveforms contain some low-frequency noise that can shift the PMT baseline locally by a small amount. The MAU computes these small local variations of the baselines, which allows for a better peak finding algorithm."`

Reply:
I have rewritten this part because it is not accurate to describe this whole thing as a MAU subtraction. In the process, MAU is defined the first time the term is used. Let me know what you think.